packages/serverless-framework-schema/json/aws/common/authorizer.json

```diff
@@ -47,12 +47,12 @@
                 "type": {
                     "type": "string",
                     "enum": [
-                        "token",
-                        "request",
-                        "cognito_user_pools"
+                        "TOKEN",
+                        "REQUEST",
+                        "COGNITO_USER_POOLS"
```

Comment:
@pavelvlasov Should I add the following values as well over here as mentioned by @FerroO2000 in https://github.com/threadheap/serverless-ide-vscode/issues/108#issuecomment-731662282?

* NONE
* AWS_IAM
* CUSTOM
* AWS_CROSS_ACCOUNT_IAM
* JWT

Out of these I was only able to find the source for `JWT` in the [documentation for `AWS::ApiGatewayV2::Authorizer`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-authorizer.html#cfn-apigatewayv2-authorizer-authorizertype).

Reply:
Probably it makes sense to put all these values and allow to put a custom string. I'll merge PR as is and tweak it from here.